os-partition-txnrecovery/configure.sh

```diff
@@ -1,11 +0,0 @@
-#!/bin/sh
```

Comment:
Just to mention I can't see where the call to this used to be?

Reply:
the script to be applied for particular module is defined  in descriptor `module.yaml` like here https://github.com/jboss-openshift/cct_module/blob/35018c1213fd064124b0464ac8703fd506f14548/os-partition-txnrecovery/module.yaml